### ISSUE TITLE
Malformed UTF-8 Characters when adding a node

### ISSUE THREAD

**Issue by nflorentin** [verbb/navigation#143]
Hi,

On a fresh install, everytime I want to add a node of type "entry" to a navigation or if there is already a node of type "entry" in the navigation, I got an error (red growl at the top of the window) and the following error is logged in the web.log file:

```
2020-03-19 16:20:09 [-][1][f9fhkibdg1tmj818q9asreqhoc][error][yii\base\InvalidArgumentException] yii\base\InvalidArgumentException: Malformed UTF-8 characters, possibly incorrectly encoded. in /Users/nicolas/www/craftcms/linksium/vendor/yiisoft/yii2/helpers/BaseJson.php:133
Stack trace:
#0 /Users/nicolas/www/craftcms/linksium/vendor/yiisoft/yii2/helpers/BaseJson.php(67): yii\helpers\BaseJson::handleJsonError(5)
#1 /Users/nicolas/www/craftcms/linksium/vendor/yiisoft/yii2/web/JsonResponseFormatter.php(119): yii\helpers\BaseJson::encode(Array, 320)
#2 /Users/nicolas/www/craftcms/linksium/vendor/yiisoft/yii2/web/JsonResponseFormatter.php(104): yii\web\JsonResponseFormatter->formatJson(Object(craft\web\Response))
#3 /Users/nicolas/www/craftcms/linksium/vendor/yiisoft/yii2/web/Response.php(1069): yii\web\JsonResponseFormatter->format(Object(craft\web\Response))
#4 /Users/nicolas/www/craftcms/linksium/vendor/craftcms/cms/src/web/Response.php(199): yii\web\Response->prepare()
#5 /Users/nicolas/www/craftcms/linksium/vendor/yiisoft/yii2/web/Response.php(337): craft\web\Response->prepare()
#6 /Users/nicolas/www/craftcms/linksium/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Response->send()
#7 /Users/nicolas/www/craftcms/linksium/web/index.php(21): yii\base\Application->run()
```

### Steps to reproduce 

I don't know how to give instructions to reproduce as I only installed the plugin, created a new navigation and added a node.

### Additional info

If I manually refresh the page, I can see that the node is correctly added. So I think the problem is in the response of the ajax call...

I see malformed characters in the html, But I'm not sure if this can be related to the error I got.
Example: `<span class="node-type-entr�e">entr�e</span>`
The function [getNodeTypeLabel](https://github.com/verbb/navigation/blob/72f79624659d89293245ee6ce26cb86416f3f071/src/elements/Node.php#L342) (which is used in twig templates) uses the function displayName() of the element, which can return string with accents.

- Plugin version: 1.3.6
- Craft version: Craft Pro 3.4.9

Thanks in advance for your help!

**Comment by engram-design** [comment]
Can you let me know if this is a multi-site? What language is your control panel in as well? Sounds like it might be in French?

What browser were you using as well? Can you inspect the Network tab in developer tools (in Chrome), just to see what the response is from the server?

Even switching the CP language for me to French doesn't seem to replicate the issue, which I thought it might.

**Comment by engram-design** [comment]
Cancel that, just _somehow_ got this to replicate! Fixed in [1.3.7](https://github.com/verbb/navigation/releases/tag/1.3.7)

**Comment by nflorentin** [comment]
Ops, sorry for the missing details about the bug. Glad you found out the bug. I tested it and it is working well. Thanks you for your reactivity!
